packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -243,13 +243,13 @@ To activate the Inline Hook, you first need to register your external service en
 
 You then need to associate the registered Inline Hook with your Profile Enrollment policy. (For information on configuring a Profile Enrollment policy, see [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm).)
 
-1. Go to **Security > Profile Enrollment**
+1. Go to **Security > Profile Enrollment**.
 
 1. Click the pencil icon to edit the policy and associate it with your Registration Inline Hook.
 
 1. In **Enrollment Settings**, click the More Options icon and then select **Edit**. If not already selected, click **Allowed** in the **For new users** section.
```

Comment:
- consistency with "pencil icon" and "More Options icon"
- "click **Allowed**" => "select the **Allowed** check box" ?

Reply:
Will match what I added for the guide page. These are the same, generic instructions. Thanks.